### DISCUSSION TITLE
posts/022-license-exchange/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#8] [Announcements]
# posts/022-license-exchange/

換護照的難度令人望而卻步，所以快來試試換駕照！雖然換完還是買不起車，，，

**Comment by PermeationLoop** [comment]
不知道爲什麼許多歐洲國家都不承認美國的駕照🤔目前看來最好用的駕照還得是歐洲國家的（法國

**Comment by PermeationLoop** [comment]
### 補充一則關於香港駕照『免試增加』准駕車型的手續方法

上文說過了，俺的香港駕照是用日本的換的，私家車的部分有自動檔的限制．
俺想用中國的 C1 來免試解除這個限制，於是就給運輸署發了一封郵件諮詢手續如何

直接說重點吧：
走的還是一般的免試換領的流程，但是要額外提交一個「正式駕駛執照及駕駛教師執照新領、續領及加領申請表」(TD 557申請表) 表格．

並且，手續費還是，港幣，**900 塊！！**
#### 我  草，生財有道！

**Reply by PermeationLoop** [reply]
補充一則香港運輸署對於免試換領的限制：

如果你的外國駕照是你在外國工作和讀書連續停留 6 個月期間簽發的，則可以免試換領；
否則就必須在簽發了的 5 年之後纔能換領。

這就導致了俺的一個羣友放棄了去大陸考電單車的夢想，轉而在香港本地報名電單車班。花費金幣 6000+ 和時間半年以上 😔

**Comment by PermeationLoop** [comment]
剛跟一個臺灣人騎車出去買東西，他說臺灣駕照屬於是點擊就送，考個摩托車甚至只需要三百四百人民幣；然後來到日本這邊換證也是點擊就換．所以日本這邊交通制度就突出一個父慈子孝 + 收割韭菜☝🏻️🤮

**Comment by PermeationLoop** [comment]
**悲報，日本駕照換領車門逐漸關緊**

<img width="568" height="717" alt="image" src="https://github.com/user-attachments/assets/c819abc4-136d-4baf-b766-1d49ffc4d094" />

一直以來，爲了能讓沒有身分的日僑能夠換領駕照，日本留了一個不需要長期簽證就能申請換領的法律漏洞。然而，這個漏洞被老中大規模濫用，甚至催生出旅遊換證產業鏈……而這次收緊之後，外國人必須持有長期簽證纔能申請換領，並且筆試答題難度，路考難度大大增加。。。😭

雖然說本次收緊看上去是爲了迎合日本國內「外國人車禍多，要管，多管！憑什麼我們那麼難考外國人隨便就能換！」的 fuck cars 民意，但是實際原因很可能是爲了解決老中的大量申請帶來的行政資源的枯竭，即，換領手續的預約排號，直到兩個月之後都沒有名額，導致有真正需要的人無法進行申請。

老中啊你如蝗蟲出征，寸草不生。雖然俺平時天天噴你日 fuck cars 露露思想橫行，但是這個政策一出來，俺反而不知道是應該拍手叫好還是該悲傷了。你們一般通過老中爲了拿一張 IDP，先後搞黃了香港和韓國濟州島的換領手續，現在就連考核非常嚴格，通過率奇低的日本，都要被你們搞到關門了。你們想要 IDP，應該做的事情是推動你們的國家去改善道路交通執法，把交通慣例對齊國際慣例，把國際做法融入駕考培訓，最後推動你國加入 1968 條約，而不是去迫害其他國家。不過鑑於你國的政治狀況，你的聲音保證傳不到上面，所以我對此感到十分悲觀。

身爲一般人，我們還能做甚麼？這是一個囚徒困境，既然國家指望不上，那你只能做出你當前最好的選擇——在最後一點有限的時間裏，多那幾個國家地區的身分，多換幾個國家地區的駕照。😢

**Reply by PermeationLoop** [reply]
悲報，日本不僅要收緊免試換領，還要禁止非居住的外國人對自己合法取得的駕照進行到期換新。

無論你砸了多少錢進去考試，一到期全部打水漂。
所以我纔會在 [《The Invincible Japanese Driver on Chinese Roads》](https://ex-tasty.com/posts/040-challenge-chinese-road/#introduction-of-japanese-driver) 一文中將日本駕照稱爲 Garbage。你就是一個垃圾，一個一個一個瞧不起人的廢紙。

你日向你法我笑國家更進了一步。
